pkg/features/kube_features.go

```diff
@@ -33,6 +33,8 @@ const (
 	//
 	// Enables partial admission.
 	PartialAdmission featuregate.Feature = "PartialAdmission"
+	// Enables cluster queue visibility.
+	ClusterQueueVisibility featuregate.Feature = "ClusterQueueVisibility"
```

Comment:
```suggestion
	QueueVisibility featuregate.Feature = "QueueVisibility"
```

@mimowo @stuton IIRC, this featureGate name is `QueueVisibility`.  Or does this feature focus only on the visibility for ClusterQueue?

https://github.com/kubernetes-sigs/kueue/blob/c2ff1bd87e3674e047969884dfa9dd773bfb0b0d/keps/168-pending-workloads-visibility/kep.yaml#L25-L26

Reply:
The proposal in KEP is to have a single feature gate `QueueVisibility` that would guard the entire feature (for LocalQueue & ClusterQueue), so I suggest using this name.

Reply:
renamed https://github.com/kubernetes-sigs/kueue/pull/1139/commits/0284c1110a8b841d28b80befee3fc8869cc6fa09